models/CommentManager.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+class CommentManager extends Model{
+    
+    //FONCTIONS POUR LE FRONTEND
+    // Fonction qui permet de récupérer les commentaires visibles d'un article (seen = 1)
+     function get_comments_blog(){
+        $req = $this->db->query("SELECT * FROM comments WHERE article_id = '{$_GET['id']}' AND seen = 1 ORDER BY date DESC");
+
+        $responses=[]; // Obligé d'initialisé avant  la boucle sinon cela ne fonctionne pas.
+        
+        while($row = $req->fetch()){   
+            $response   = new Comment();
+            $response   ->setName($row['name']);
+            $response   ->setDate($row['date']);
+            $response   ->setComment($row['comment']);
+            $response   ->setArticleId($row['article_id']);
+            $response   ->setSeen($row['seen']);
+                        
+            $responses[] = $response;
+        }
+        return $responses;
+        }
+        // Fonction permettant de vérifier que les données envoyées pat l'utilisateur sont bonnes.
+        function form_comment_verification(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method form_comment_verification() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)